lib/xrp/XrpRpc.js

```diff
@@ -0,0 +1,210 @@
+const RippleAPI = require('ripple-lib').RippleAPI;
+const rippleBinaryCodec = require('ripple-binary-codec');
+const promptly = require('promptly');
+const util = require('util');
+const EventEmitter = require('events');
+
+const passwordPromptAsync = util.promisify(promptly.password);
+
+class XrpRpc {
+  constructor(config) {
+    this.config = config;
+    const {
+      rpcPort,
+      host,
+      protocol,
+      address
+    } = config;
+    const connectionString = `${protocol}://${host}:${rpcPort}`;
+    this.rpc = new RippleAPI({
+      server: connectionString
```

Comment:
We may want to handle the possibility of http/s protocol. Maybe depending on the protocol we decide whether to use `server` for wss or `proxy` for http/s